scripts/package.lua

```diff
@@ -40,7 +42,7 @@
 -- Make sure I've got what I've need to be happy.
 --
 
-	local required = { "git", "make", "gcc", "premake5", "zip" }
+	local required = { "git", "make", "gcc", "zip" }
```

Comment:
This is strange, but if premake5 is present here, package.lua never works for me. Problem exists in all environments I try.